homeassistant/components/sensor/xbox_live.py

```diff
@@ -34,15 +34,25 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     api = xbox_api.XboxApi(config.get(CONF_API_KEY))
     devices = []
 
+    # request personal profile to check api connection
+    profile = api.get_profile()
+    if profile.get('error_code', None) is not None:
```

Comment:
You can remove `None`. It's the [default](https://docs.python.org/3.7/library/stdtypes.html#dict.get).

Reply:
Thank you for your feedback. I've implemented the requested change.